src/components/Forms/Category-form.tsx

```diff
@@ -53,8 +44,7 @@ const CategoryForm:React.FC<CategoryFormProps> = ({dispatch, categoriesLength, h
        onSubmit={(values, { setSubmitting }) => {
```

Comment:
its a bad  practice to define anononymous functions inside render of components, cause it  creates new functions each render, define something like onSubmitting in Component itself.

Also i recommend, to use withFormik hook, against Formi component

Reply:
Rewrite code with HOK WithFormik.